resources/conf/log_config.toml

```diff
@@ -43,26 +43,26 @@ logType = "text" # LogTypes can be "text" or "json"
 secondaryLogFormat = ""
 
 [accessLogs.jSONFormat]
-t = "%START_TIME%"
+time = "%START_TIME%"
 gwHost = "%DYNAMIC_METADATA(envoy.filters.http.ext_authz:originalHost)%"
 host = "%REQ(:AUTHORITY)%"
-m = "%REQ(:METHOD)%"
-gwPath = "%DYNAMIC_METADATA(envoy.filters.http.ext_authz:originalPath)%"
-reqPath = "%REQ(:PATH)%"
-rtName = "%ROUTE_NAME%"
-prtcol = "%PROTOCOL%"
-resCd = "%RESPONSE_CODE%"
-resCdInf = "%RESPONSE_CODE_DETAILS%"
-resFlg = "%RESPONSE_FLAGS%"
-usrAgt = "%REQ(USER-AGENT)%"
+method = "%REQ(:METHOD)%"
+apiPath = "%DYNAMIC_METADATA(envoy.filters.http.ext_authz:originalPath)%"
+upstrmPath = "%REQ(:PATH)%"
+rName = "%ROUTE_NAME%"
+prot = "%PROTOCOL%"
+respCode = "%RESPONSE_CODE%"
+respCodeDtls = "%RESPONSE_CODE_DETAILS%"
+respFlag = "%RESPONSE_FLAGS%"
+ua = "%REQ(USER-AGENT)%"
 reqId = "%REQ(X-REQUEST-ID)%"
-fwdFor = "%REQ(X-FORWARDED-FOR)%"
-upsHost = "%UPSTREAM_HOST%"
-bytRec = "%BYTES_RECEIVED%"
-bytSnt = "%BYTES_SENT%"
-du = "%DURATION%"
-upsSvcTime = "%RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)%"
-reqTxDu = "%REQUEST_TX_DURATION%"
-resTxDu = "%RESPONSE_TX_DURATION%"
-reqDu = "%REQUEST_DURATION%"
-resDu = "%RESPONSE_DURATION%"
+xff = "%REQ(X-FORWARDED-FOR)%"
+upstrmHost = "%UPSTREAM_HOST%"
+bytesRecv = "%BYTES_RECEIVED%"
+bytesSent = "%BYTES_SENT%"
+dur = "%DURATION%"
+upstrmSvcTime = "%RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)%"
+reqTxDur = "%REQUEST_TX_DURATION%"
```

Comment:
Is it possible to clarify more the term `Tx` value ?

Reply:
I guess it is a common abbreviation used for transmission and for transactions.